_posts/2016-07-01-newsletter.md

```diff
@@ -0,0 +1,254 @@
+automated change-aware linter, lua-VimL PR. He gave careful attention to
+internals such as `msgpackparse()`/`msgpackdump()`, `string()`, and completely
+re-wrote `json_decode()`/`json_encode()` support (incl. granular error messages
+and comprehensive test coverage).
+
+
+Release strategy
+----------------
+
+### 0.1.x releases
+
+A few months ago we announced our first release, `0.1.`. In the months since
+then, we've streamlined our release process (versioning, tagging, changelogging,
+announcement-making, booty-shaking).
+
+I consider these releases reasonably stable, though we still have some bugs that
+we'd rather not have. With `0.2`, Windows will become a first-class target.
+
+Some question the lack of a `1.0` sticker. We follow
+[semver](https://semver.org), the recommendations there explain the status that
+we want to signal with the `0.x` version designations. We don't want to declare
+`1.0` until we have some features like providers, UIs, and the remote API
+designed to a spec that we want to commit to supporting for the (very) long
+term.
+
+### OS Packages
+
+More OS packages are appearing. Thanks to @jamessan, @fwalch and others for
+building packages for their favorite systems and working with us at
+neovim/neovim to address inevitable compiler/platform quirks that come from
+supporting dozens of targets.
+
+Progress
+--------
+
+Each Neovim PR now builds against 12(!) different systems. Thanks to
+@jszakmeister, @fwalch and @ZyX-I for building a sophisticated pipeline.
+Reliable and meaningful CI/automation is central to our strategy, "Move Fast and
+Don't Break Things™".
+
+We did break some things though. Many of those regressions have been fixed
+thanks to @oni-link's work, and many many others. Two years ago when we forked
+Vim, there was an unstable period; that gap has become smaller and smaller.
+Each of regression fix is covered by integration tests.
+
+In February a user asked if Neovim is in "maintenance mode"; my answer was
+"absolutely": we had technical debt to pay down.
+
+To pay down that debt we very much want the cost and investment to be
+a _one-time_ cost. So tons of work has been put into (1) test infrastructure,
+(2) automation of administrative tasks (like merging Vim patches, linting code),
+and (3) documentation for users and contributors (answering questions is
+important, so it can't be ignored--but it's also time-consuming; spending time
+to clearly write down decisions and references will free up time for maintainers
+to get back to working on new features).
+
+(TODO: laundry list of the tons of work we've done in the last 2 years, which
+wasn't really addressed by previous newsletters and is obviously unknown to most
+people based on reddit conversations)
+
+- `:terminal`
+- remote API
+- first-class XDG support
+- better defaults
+
+### New features
+
+- `buffer_add_highlight` API method. `:help api-highlights` position-based
```

Comment:
maybe not mention `buffer_add_highlight` by name as that name is soon deprecated. The `:help` reference is good enough and most (all?) existing plugins are not using that name directly but e.g. `buf.add_highlight` in python (which will not change).